README.md

```diff
@@ -241,9 +241,9 @@ AppAuth supports three options for dependency management.
    With [CocoaPods](https://guides.cocoapods.org/using/getting-started.html), add the following line to
    your `Podfile`:
 
-       pod 'AppAuth', '>= 0.94'
+       pod 'AppAuth', '1.0'
 
-   Then run `pod install`. Note that version 0.94 is the first of the library to support iOS 11.
+   Then run `pod install`.
```

Comment:
FYI, all instructions about linking on both paltforms are obsolete in RN 0.60 if the library works with autolinking. All you need to document now is how to use with Carthage or anything that is not CocoaPods or manual link

Reply:
You're right, I'll update the instructions. Thanks for all your help on this!